draft-ietf-quic-multipath.md

```diff
@@ -635,6 +635,9 @@ data packets, including 0-RTT packets, using the initial Connection ID with
 sequence number 0 after the handshake concluded.
 
 ACK_MP frames (defined in {{ack-mp-frame}}) can be returned on any path.
+If the ACK_MP is prefered to be sent on the same path as the acknowledged
+packet (see {{compute-rtt}} for further guidance), it can be beneficial to bundle an MP_ACK frame
+with the PATH_RESPONSE frame during path validation.
```

Comment:
```suggestion
packet (see {{compute-rtt}} for further guidance), it can be beneficial
to bundle an MP_ACK frame with the PATH_RESPONSE frame during
path validation.
```